go/pkg/client/cas.go

```diff
@@ -597,6 +601,9 @@ func (c *Client) ResourceNameWrite(hash string, sizeBytes int64) string {
 // GetDirectoryTree returns the entire directory tree rooted at the given digest (which must target
 // a Directory stored in the CAS).
 func (c *Client) GetDirectoryTree(ctx context.Context, d *repb.Digest) (result []*repb.Directory, err error) {
+	if d.SizeBytes == 0 && d.Hash == digest.Empty.Hash {
+		return nil, nil
```

Comment:
Wait, is that correct? Shouldn't this return an empty Directory proto?

Reply:
It's returning a slice of directory protos, so nil is pretty equivalent to an empty slice here.

Reply:
Yes, but I mean: shouldn't it return a slice containing one item, `[]*repb.Directory{&repb.Directory{}}`, instead? An empty digest maps to an empty directory.